### DISCUSSION TITLE
How to get the last verison for new UI ?

### DISCUSSION THREAD

**Discussion by lufoaye** [community/community#18778] [Other Feature Feedback, Questions, & Ideas]
Hello, i hava a project is working base on LVGLBuider, but i saw some infos for 1.0.4 version, can you show me or send the code for 1.0.4 to me , i can match some new code for new verison.

**Comment by thomscoder** [comment]
Hi,
I guess you're talking about [this](https://github.com/CURTLab/LVGLBuilder). You can open an Issue there to ask directly to the mantainers.
